site/docs/develop/spec.md

```diff
@@ -1,80 +1,180 @@
+ensure that schema evolution works as expected.
 
-There are several expections of a version store. Some are listed below
+The Nessie `IcebergTable` object passed to Nessie in a [_Put operation_](#put-operation) therefore
+consists of
 
-!!! warning
-    still missing some conditions
-    
-### Conflict Resolution
-The API passes an ‘expectedHash’ parameter when it modifies a key. This is the commit that the client thinks is the most up to date (its HEAD). A backend will check to see if the key has been modified since that ‘expectedHash' and if, it will reject the requested modification with FAILED_PRECONDITION. This is basically an optimistic lock that accounts for the fact that the commit hash is global and nessie branch could have moved on from ‘expectedHash’ without modifying the key in question.
+1. the pointer to the Iceberg _table metadata_ and
+2. the ID of the _Iceberg snapshot_ within the Iceberg _table metadata_.
+
+The pointer to the Iceberg table is recorded as _Global State_ and the ID of the Iceberg snapshot
+is recorded within the _Put operation_ in a Nessie commit.
+
+!!! note
+    This model puts a strong restriction on the Iceberg table. All metadata JSON documents must be
+    stored and none of the built-in iceberg maintenance procedures can be used. There are
+    potentially serious issues regarding schema migrations in this model as well. Therefore, the
+    Iceberg table spec should be considered subject to change in the near future.
+
+#### Delta Lake Table
 
-The reason for this condition is to behave like a ‘real’ database. You shouldn’t have to update your reference before transacting on tableA because it just happened to update tableB whilst you were preparing your transaction.
+The state of a Delta Lake Table is represented using the Delta Lake Table attributes
+`metadataLocationHistory`, `checkpointLocationHistory` and `lastCheckpoint`.
 
-## Tiered Version Store
+Delta Lake Tables are tracked without a _Global State_ in Nessie, i.e. those three attributes are
+recorded within the [_Put Operation_](#put-operation) of a Nessie commit.
+
+#### View
+
+The state of an SQL view is represented using the attributes
+`sqlText` and `dialect` (currently one of `HIVE`, `SPARK`, `DREMIO`, `PRESTO`).
+
+SQL views are tracked without a _Global State_ in Nessie, i.e. those three attributes are
+recorded within the [_Put Operation_](#put-operation) of a Nessie commit.
+
+## Operations in a Nessie commit
+
+Each Nessie commit carries one or more operations. Each operation contains the Content Key plus and
+comes in one of the following variations.
+
+### Put operation
+
+A _Put operation_ modifies the state of the included Content object. It must contain the Content
+object and, if the Content type tracks [_Global State_](#on-reference-state-vs-global-state), also
+the _expected contents_. The _expected contents_ attribute can be omitted, if the Content object
```

Comment:
Could you add a link to the conflict resolution paragraph (below) regarding the use of _expected contents_?